cluster-provision/k8s/1.18/manifests/cni.diff

```diff
@@ -0,0 +1,22 @@
+--- calico.3.15.1.yaml	2020-07-26 11:55:32.000000000 +0300
++++ cni_updated.yaml	2020-07-26 12:30:39.000000000 +0300
+@@ -34,6 +34,9 @@
+           "ipam": {
+               "type": "calico-ipam"
+           },
++          "container_settings": {
++              "allow_ip_forwarding": true
++          },
+           "policy": {
+               "type": "k8s"
+           },
+@@ -3709,6 +3712,9 @@
+           effect: NoSchedule
+       serviceAccountName: calico-kube-controllers
+       priorityClassName: system-cluster-critical
++      securityContext:
```

Comment:
I see we had this in the previous cni.yml, but do we still need it with the latest Calico version?

Reply:
i didnt try without, i can try if needed, unless something set it, i think we will need, else stuff might fail

Reply:
I see it was intentionally added in https://github.com/kubevirt/kubevirtci/pull/321/files, so we likely need to keep it.